HelpSource/Classes/Array.schelp

```diff
@@ -365,10 +365,11 @@ Routine {
 	a.fork; \doneAll.postln;
 }.play;
 )
+::
 
-poll(trig, label, trigid)
+method::poll(trig, label, trigid)
 apply an array of Poll units to an array of UGens (see those helpfiles for more details).
-
+code::
```

Comment:
you can also remove the `s.boot;`. A booted server is assumed to be the case anyway.

Reply:
done.